Sources/md-babel/Execute/RunProcess.swift

```
@@ -169,7 +169,7 @@ struct RunProcess {
 				// explicitly.
 				//
 				// We apply the same logic to `readIO` below.
-				try! inputPipe.fileHandleForWriting.close()
+				try? inputPipe.fileHandleForWriting.close()
```

Especially since a CLI is run synchronously to exit immediately, a crash is better than silently dropping the error and moving on.

While syntactically, this is a 'fix' for a force try, it leaves the application in an invalid state and from here on out there's undefined behavior.

So please rather remove the force try rule from swift-format -- at least as an error.

---

Makes sense. I also removed `NeverForceUnwrap`.